src/twisted/conch/interfaces.py

```diff
@@ -150,8 +150,9 @@ def openFile(filename, flags, attrs):
 
         @param filename: a string representing the file to open.
 
-        @param flags: an integer of the flags to open the file with, ORed together.
-        The flags and their values are listed at the bottom of this file.
+        @param flags: an integer of the flags to open the file with, ORed
+        together.  The flags and their values are listed at the bottom of
+        L{twisted.conch.ssh.filetransfer} as FXF_*.
```

Comment:
Unfortunately these don't seem to show up in our API docs: https://twistedmatrix.com/documents/current/api/twisted.conch.ssh.filetransfer.html

![image](https://user-images.githubusercontent.com/43662/66729581-30a4de80-ee01-11e9-92f5-f82f88c62a7f.png)

Still, the "view source" link gets you to the right place, so this is a useful improvement. I filed https://twistedmatrix.com/trac/ticket/9714 to properly document these constants.

Reply:
`@var` annotations at the module level is what it takes to get these to show up.